build/debian-base/Makefile

```diff
@@ -18,30 +18,30 @@ REGISTRY ?= gcr.io/google-containers
 IMAGE ?= debian-base
 BUILD_IMAGE ?= debian-build
 
-TAG ?= 0.2
+TAG ?= 0.3
 
 TAR_FILE ?= rootfs.tar
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
 QEMUVERSION=v2.9.1
 
 ifeq ($(ARCH),amd64)
-	BASEIMAGE?=debian:jessie
+	BASEIMAGE?=debian:stretch
 endif
 ifeq ($(ARCH),arm)
-	BASEIMAGE?=arm32v7/debian:jessie
+	BASEIMAGE?=arm32v7/debian:stretch
 	QEMUARCH=arm
 endif
 ifeq ($(ARCH),arm64)
-	BASEIMAGE?=arm64v8/debian:jessie
+	BASEIMAGE?=arm64v8/debian:stretch
 	QEMUARCH=aarch64
 endif
 ifeq ($(ARCH),ppc64le)
-	BASEIMAGE?=ppc64le/debian:jessie
+	BASEIMAGE?=ppc64le/debian:stretch
 	QEMUARCH=ppc64le
 endif
 ifeq ($(ARCH),s390x)
-	BASEIMAGE?=s390x/debian:jessie
+	BASEIMAGE?=s390x/debian:stretch
```

Comment:
I'm having trouble building the base image for s390x. It seems the `/sbin/ldconfig` binary in `debian:stretch` is either broken or doesn't work with qemu:
```
Processing triggers for libc-bin (2.24-11+deb9u1) ...
/var/lib/dpkg/info/libc-bin.postinst: 49: /var/lib/dpkg/info/libc-bin.postinst: ldconfig: Exec format error
/var/lib/dpkg/info/libc-bin.postinst: 49: /var/lib/dpkg/info/libc-bin.postinst: ldconfig: Exec format error
dpkg: error processing package libc-bin (--configure):
 subprocess installed post-installation script returned error exit status 2
Errors were encountered while processing:
 libc-bin
```
@gajju26

Reply:
From `debian:jessie`:
```
root@219babe053eb:/# file /sbin/ldconfig
/sbin/ldconfig: POSIX shell script, ASCII text executable
root@219babe053eb:/# file /sbin/ldconfig.real
/sbin/ldconfig.real: ELF 64-bit MSB executable, IBM S/390, version 1 (SYSV), statically linked, for GNU/Linux 2.6.32, BuildID[sha1]=0489f9fffcaa0413e00565d6b53344244f686103, stripped
```

From `debian:stretch`:
```
root@1343b24ec59d:/# file /sbin/ldconfig
/sbin/ldconfig: ELF 64-bit MSB executable, IBM S/390, version 1 (GNU/Linux), statically linked, for GNU/Linux 3.2.0, BuildID[sha1]=90b64604014aafac9c1a0623b1cf447281d1a382, stripped
```

I'm guessing that `GNU/Linux` vs `SYSV` difference might be the issue? Not sure how to resolve, though.

Reply:
@ixdy i submitted an issue upstream. When I download the various released qemu-static binaries from the 2.9.1 tag, their release version tell me 2.9.0.

Reply:
I also created a fix to mask off the e_ident ELF flags from the binfmt 390x files in the qemu-user-static project.

I will update this issue once the patch gets merged upstream in the qemu-user-static project.

https://github.com/multiarch/qemu-user-static/pull/31

Reply:
Tested a local build with this upstream patch. The s390x image successfully gets built.